converter/internal/prometheusconvert/testdata/consul.river

```diff
@@ -0,0 +1,45 @@
+prometheus.remote_write "default" {
+	external_labels = {}
```

Comment:
I don't think this should be set. 

It looks like we always return a non-nil map for this: 

https://github.com/grafana/agent/blob/a95a2bd2eb68a0b6ef692d29f21f89dd1b64bb80/converter/internal/prometheusconvert/remote_write.go#L23C18-L23C64

I think it should be nil if the map is empty so this doesn't print.